modules/openapi-generator/src/main/resources/dart2/class.mustache

```diff
@@ -1,105 +1,161 @@
-class {{classname}} {
+class {{{classname}}} {
+  {{{classname}}}({
+  {{#vars}}
+    {{#isNullable}}{{#required}}@required {{/required}}this.{{{name}}},{{/isNullable}}{{^isNullable}}{{#defaultValue}}this.{{{name}}} = {{#isContainer}}const {{/isContainer}}{{{defaultValue}}}{{/defaultValue}}{{^defaultValue}}{{#required}}@required {{/required}}this.{{{name}}}{{/defaultValue}},{{/isNullable}}
+  {{/vars}}
+  });
+
   {{#vars}}
   {{#description}}/// {{{description}}}{{/description}}
   {{^isEnum}}
-  {{^defaultValue}}{{{dataType}}} {{name}};{{/defaultValue}}{{#defaultValue}}{{{dataType}}} {{name}} = {{defaultValue}};{{/defaultValue}}
+  {{{dataType}}} {{{name}}};
   {{/isEnum}}
   {{#isEnum}}
-  {{#allowableValues}}
-  {{#min}} // range from {{min}} to {{max}}{{/min}}{{classname}}{{{enumName}}} {{name}}{{#required}} = {{classname}}{{{enumName}}}._internal({{{defaultValue}}}){{/required}}{{^required}}{{/required}};
-  {{/allowableValues}}
+    {{#isContainer}}
+      {{#isListContainer}}
+  List<{{{classname}}}{{{enumName}}}> {{{name}}};
+      {{/isListContainer}}
+      {{#isMapContainer}}
+  Map<String, dynamic> {{{name}}};
+      {{/isMapContainer}}
+    {{/isContainer}}
+    {{^isContainer}}
+      {{#allowableValues}}
+  {{#min}} // range from {{{min}}} to {{{max}}}{{/min}}{{{classname}}}{{{enumName}}} {{{name}}}{{#required}} = {{{classname}}}{{{enumName}}}._({{{defaultValue}}}){{/required}}{{^required}}{{/required}};
+      {{/allowableValues}}
+    {{/isContainer}}
   {{/isEnum}}
+
+  {{/vars}}
+  @override
+  bool operator ==(Object other) => identical(this, other) || other is {{{classname}}} &&
+  {{#vars}}
+     other.{{{name}}} == {{{name}}}{{^-last}} &&{{/-last}}{{#-last}};{{/-last}}
   {{/vars}}
 
-  {{classname}}({
+  @override
+  int get hashCode =>
   {{#vars}}
-    {{#required}}@required this.{{name}}{{/required}}{{^required}}this.{{name}}{{#defaultValue}} = {{defaultValue}}{{/defaultValue}}{{/required}},
+    {{#isNullable}}({{{name}}}?.hashCode ?? 0){{/isNullable}}{{^isNullable}}{{{name}}}.hashCode{{/isNullable}}{{^-last}} +{{/-last}}{{#-last}};{{/-last}}
   {{/vars}}
-  });
 
   @override
   String toString() {
-    return '{{classname}}[{{#vars}}{{name}}=${{name}}, {{/vars}}]';
+    final sb = StringBuffer();
```

Comment:
Is there a specific reason for this change? I adds a lot of unnecessary clutter in my opinion

Reply:
Just because of the final `, ` -- I like my code tidy even in the smallest details. After all, this is a template and rarely does anybody venture here.

But if you want this to be back to what it was, it's not a problem.

Reply:
You're absolutely right, fixed now.